packages/@react-stately/virtualizer/src/Virtualizer.ts

```diff
@@ -392,6 +392,10 @@ export class Virtualizer<T extends object, V, W> {
     return layoutInfo.key;
   }
 
+  cancelRelayout() {
```

Comment:
Can we make this more general? e.g. `willUnmount`? That way if there's other cleanup we need to do here later we could add it in the same method.